app/controllers/sync_controller.rb

```diff
@@ -0,0 +1,19 @@
+require 'legacy_data_sync'
+
+class SyncController < ApplicationController
```

Comment:
nitpicking here, but maybe namespace all our API stuff?

e.g. `API::Datasets::SynchronizationController`, under `app/controllers/api/datasets/synchronization_controller.rb`

Reply:
Overkill, especially since this is a temporary endpoint

Reply:
No longer required